orangecontrib/timeseries/widgets/owyahoofinance.py

```diff
@@ -26,57 +28,45 @@ class Outputs:
     date_from = settings.Setting(
         (datetime.now().date() - timedelta(5 * 365)).strftime(PY_DATE_FORMAT))
     date_to = settings.Setting(datetime.now().date().strftime(PY_DATE_FORMAT))
-    symbols = settings.Setting(['AMZN', 'AAPL', 'GOOG', 'FB', 'SPY', '^DJI', '^TNX'])
+    symbols = settings.Setting(
+        ['AMZN', 'AAPL', 'GOOG', 'FB', 'SPY', '^DJI', '^TNX'])
 
     want_main_area = False
     resizing_enabled = False
 
     class Error(widget.OWWidget.Error):
-        download_error = widget.Msg('Failed to download data (HTTP Error {}). '
-                                    'Wrong stock symbol?')
+        download_error = widget.Msg('Failed to download data (HTTP Error {}).\n'
```

Comment:
Regardless of the type of error (wrong symbol or no internet), I always get "HTTP Error -1", which is quite uninformative. Perhaps we can just remove the formatted string?

Reply:
You're right. I remove the part in parentheses; HTTP is also useless, the user doesn't need to care about the protocol.